packages/fxa-auth-server/.vscode/launch.json

```diff
@@ -87,16 +87,15 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Arch 2.0 with Coverage",
+      "name": "Current File with Coverage",
       "program": "${workspaceFolder}/node_modules/nyc/bin/nyc.js",
       "args": [
         "--reporter=lcov",
         "${workspaceFolder}/node_modules/mocha/bin/_mocha",
         "--timeout",
         "999999",
         "--colors",
-        "${workspaceFolder}/test/local/payments/stripe.js",
-        "${workspaceFolder}/test/local/routes/subscriptions.js",
+        "${workspaceFolder}/${relativeFile}",
```

Comment:
Maybe add a new configuration instead replacing a current one?

Reply:
The current one was one that I had created when adding the test coverage for for the new subscription routes. It was requested that I make the test more flexible once I had completed those tasks, so that it could be more useful to everyone.

Reply:
Ah, ok.  👍